ModelCatalogueGenomicsPlugin/test/integration/org/modelcatalogue/gel/export/AbstractRareDiseasesExporterSpec.groovy

```diff
@@ -82,11 +82,7 @@ class AbstractRareDiseasesExporterSpec extends IntegrationSpec {
 
                                             if (createPhenotypes) {
                                                 for (int k in 1..15) {
-                                                    dataClass name: "Phenotype ($k) name $i $j", {
-                                                        //sets a value for OBO ID in the extension map - there must be a clearer way to do this
-                                                        ContextItem contextElement = context.getContextElement(DataClass, 0)
```

Comment:
I'm sorry I haven't noticed this before. I think we've already discussed this or was it with @rickreesrr? the proper way to do it is very simple

```
 dataClass name: "Phenotype ($k) name $i $j", {
    ext "OBO ID", "HP:" + (i + j + k)
}
```